tests/forecast/test_ets.py

```diff
@@ -138,20 +137,19 @@ def _test_forecast(self):
             target_seq_index=0,
         )
         logger.info(f"MSIS = {msis:.4f}")
-        self.assertLessEqual(np.abs(msis - 101.6), 10)
```

Comment:
Can you add back these assertions with the appropriate values?

Reply:
sure, fix it